content/docs/04-clusters/1-new-clusters/02-eks.md

```diff
@@ -681,7 +715,11 @@ Palette customers can optionally disable the OIDC Associate Provider if the serv
 </InfoBox>
 
 <br />
-<br />
+<br />
+
+# Add IAM Policies At Node Level:
```

Comment:
@arvind-sys sir I am not sure if the provided information is correct

Reply:
removing the content as per the discussion with @sadysnaat and @arvind-sys